### DISCUSSION TITLE
vercel env pull is not working

### DISCUSSION THREAD

**Discussion by barahonam1** [vercel/vercel#6146] [Help]
It seems they replaced the previous system for creating environment variables in the vercel dashboard. Whenever I try to execute vercel env pull to download my latest variables for the development environment, the latest variables are not being downloaded.

Can somebody help me please?

**Comment by markozxuu** [comment]
Hi @barahonam1  👋 

I could indicate the version that use ? Try to use the last version cli which is `22.0.1`

**Comment by jsjoeio** [comment]
I'm running into the exact same issue with `vercel` `v23.0.0`

# System Information
## System:
 - OS: macOS 11.3
 - CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
 - Memory: 1.05 GB / 16.00 GB
 - Shell: 5.8 - /bin/zsh
## Binaries:
 - Node: 14.17.0 - ~/.nvm/versions/node/v14.17.0/bin/node
 - Yarn: 1.22.10 - /usr/local/bin/yarn
 - npm: 6.14.13 - ~/.nvm/versions/node/v14.17.0/bin/npm
## Managers:
 - Cargo: 1.48.0 - ~/.cargo/bin/cargo
 - Homebrew: 3.1.9 - /usr/local/bin/brew
 - pip3: 21.1.1 - /usr/local/bin/pip3
 - RubyGems: 3.0.3 - /usr/bin/gem
## Utilities:
 - Make: 3.81 - /usr/bin/make
 - GCC: 4.2.1 - /usr/bin/gcc
 - Git: 2.31.1 - /usr/local/bin/git
 - Clang: 12.0.5 - /usr/bin/clang
## Servers:
 - Apache: 2.4.46 - /usr/sbin/apachectl
## Virtualization:
 - Docker: 20.10.6 - /usr/local/bin/docker
 - Parallels: 16.1.3 - /usr/local/bin/prlctl
 - VirtualBox: 6.1.18 - /usr/local/bin/vboxmanage
## IDEs:
 - Emacs: 27.2 - /usr/local/bin/emacs
 - Nano: 2.0.6 - /usr/bin/nano
 - VSCode: 1.56.2 - /usr/local/bin/code
 - Vim: 8.2 - /usr/bin/vim
 - Xcode: /undefined - /usr/bin/xcodebuild
## Languages:
 - Bash: 5.1.4 - /usr/local/bin/bash
 - Go: 1.15.6 - /usr/local/go/bin/go
 - Perl: 5.30.2 - /usr/bin/perl
 - PHP: 7.3.24 - /usr/bin/php
 - Python: 2.7.16 - /usr/bin/python
 - Python3: 3.9.5 - /usr/local/bin/python3
 - Ruby: 2.6.3 - /usr/bin/ruby
 - Rust: 1.48.0 - /Users/jp/.cargo/bin/rustc
## Databases:
 - SQLite: 3.32.3 - /usr/bin/sqlite3
## Browsers:
 - Brave Browser: 90.1.24.85
 - Chrome: 90.0.4430.212
 - Firefox Developer Edition: 89.0
 - Safari: 14.1

# Steps to Reproduce

1. Create new Next.js project `npx create-next-app --typescript`
2. Make some changes
3. Push to GitHub `gh repo create`
4. Connect to Vercel in Dashboard -> Import Project
5. Add environment variables
6. Locally, run `npx vercel env pull`
7. Look inside `.env`

## Expected

`.env` contains environment variables set up in project

## Actual

`.env` is blank.

## Screenshots
![image](https://user-images.githubusercontent.com/3806031/120661010-bfce9200-c43c-11eb-9452-52ea0581e47e.png)
![image](https://user-images.githubusercontent.com/3806031/120661033-c52bdc80-c43c-11eb-89cc-dfc681a9b7f7.png)
![image](https://user-images.githubusercontent.com/3806031/120661081-cfe67180-c43c-11eb-84ff-daa6ec1db782.png)

**Reply by jsjoeio** [reply]
Woah..nevermind. I figured it out. I have two projects with similar names — dip.chat and dip-chat — and I accidentally linked the wrong one 🤦‍♂️
